.pre-commit-config.yaml

```diff
@@ -17,13 +17,13 @@ repos:
         types_or: [python, pyi]
         args: [--ignore-missing-imports, --scripts-are-modules]
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.7.1
+    rev: v0.8.3
     hooks:
       - id: ruff
         args: [--fix, --exit-non-zero-on-fix]
       - id: ruff-format
   - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.10
+    rev: 1.8.0
```

Comment:
Not urgent, but given that ruff implements the [`flake8-bandit` rules](https://docs.astral.sh/ruff/rules/#flake8-bandit-s), we might get rid of this hook.

Reply:
I reverted the whole update due to the observed lock file problems with setuptools_scm, but it's a good thing to note.